core/src/main/resources/schema/app/20230224135630_CB-20943_Remove_the_secrets_from_the_userdata_script_which_are_stored_in_the_database.sql

```diff
@@ -0,0 +1,22 @@
+-- // CB-20943 Remove the secrets from the userdata script which are stored in the database
+-- Migration SQL that makes the change goes here.
+
+CREATE TABLE IF NOT EXISTS userdata (
+    id                      bigserial NOT NULL,
+    stack_id                bigint,
+    accountid               VARCHAR (255),
+    coreuserdata            TEXT,
+    gatewayuserdata         TEXT,
+    PRIMARY KEY (id)
+);
+
+ALTER TABLE ONLY userdata ADD CONSTRAINT fk_userdata_stack_id FOREIGN KEY (stack_id) REFERENCES stack(id);
+CREATE SEQUENCE IF NOT EXISTS userdata_id_seq START WITH 1 INCREMENT BY 1 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE INDEX IF NOT EXISTS userdataid ON userdata (id);
```

Comment:
we usually follow the following naming convention for index: `idx_rootcert_stackid`
also you should create the index for `stack_id` instead if `id`